tests/requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytest
-pytest-mock
\ No newline at end of file
+pytest-mock
+billiard
+amqp
```

Comment:
I don't see any usage reference to amqp & billiard.

Reply:
It was required during testing, I saw failure for this dependencies so added them.

Reply:
Can you share the failure message here? Just want to understand it.

Reply:
When I was verifying now without this packages, all checks have passed. Removing this packages from test dependencies.